rudder-jetty/SOURCES/Makefile

```diff
@@ -70,6 +70,7 @@ localdepends: ./jetty7
 
 	cp ./jetty7/bin/jetty-rpm.sh ./jetty7/bin/jetty-sles.sh
 	patch -p0 -s < patches/jetty/jetty-init-sles.patch
+	patch -p0 -s < patches/jetty/jetty-init-ulimit.patch
```

Comment:
In the issue: "and EVEN with the patch, the parent bug is not corrected." how is this addressed ?